ibm/service/catalogmanagement/resource_ibm_cm_version.go

```diff
@@ -248,6 +248,107 @@ func ResourceIBMCmVersion() *schema.Resource {
 							Optional:    true,
 							Description: "Version source URL.",
 						},
+						"working_directory": &schema.Schema{
```

Comment:
Same as above we declared in schema are we using these attributes in create/update?

Reply:
Changed these to all be computed, so they arent used in create/update

Reply:
User will not able to set the metadata during onboarding process?